cli/cmd/initnodes.go

```diff
@@ -398,3 +423,96 @@ func gethInit(ctx context.Context, cfg InitConfig, dir string) error {
 
 	return nil
 }
+
+func restoreSnapshotData(ctx context.Context, homeDir string, clientName string) error {
+	log.Info(ctx, fmt.Sprintf("⌛ Restoring latest chain snapshot for %s...", clientName))
```

Comment:
Avoid SpringF in logging, we use structured logging, see go style guide

Reply:
remove emojis

Reply:
Done.